packages/client/src/exampleTypes.js

```diff
@@ -443,4 +443,9 @@ export default F.stampKey('type', {
       drilldown: 'self',
     },
   },
+  raw: {
+    reactors: {
+      filter: 'self',
```

Comment:
This should probably be others, not self.

Reply:
`filter` on a `raw` node affects other nodes in the tree by changing its filter. A self reactor tells it to update its own context (rerunning the result function). 

In your example, updating the raw node should update the terms stats node, not itself.

Reply:
To further clarify - if you’re using `raw` and expecting it’s `filter` property to work like terms stats, you’re likely to be surprised that it’s affecting the entire rest of the search and not just its own context.